core/src/main/java/com/karumi/kotlinsnapshot/core/serializers/RuntimeClassNameTypeAdapterFactory.java

```diff
@@ -0,0 +1,231 @@
+/* ktlint-disable */
```

Comment:
No need to use these comments for java files

Reply:
Fixed!